src/content/en/2020/resource-hints.md

```diff
@@ -329,57 +307,59 @@ As mentioned earlier, the [PRPL Pattern](https://addyosmani.com/blog/the-prpl-pa
 
-### Priority Hints
+### Priority hints
 
-[Priority hints](https://developers.google.com/web/updates/2019/02/priority-hints) are an API for expressing the fetch priority of a resource: high, low, or auto. They can be used to help deprioritize images (e.g. inside a Carousel), re-prioritize scripts and even help de-prioritize fetches.
+[Priority hints](https://developers.google.com/web/updates/2019/02/priority-hints) are an API for expressing the fetch priority of a resource: high, low, or auto. They can be used to help deprioritize images (e.g. inside a Carousel), re-prioritize scripts, and even help de-prioritize fetches.
 
 This new hint can be used either as an HTML tag or by changing the priority of fetch requests via the `importance` option, which takes the same values as the HTML attribute.
 
-With `preload` and `prefetch`, the priority is set by the browser depending on the type of resource. By using Priority Hints we can force the browser to change the default option.
+{# TODO(editors): Should we keep this shorter to prevent small amounts of scrolling? #}
+```html
+<!-- We want to initiate an early fetch for a resource, but also deprioritize it -->
+<link rel="preload" href="/js/script.js" as="script" importance="low">
 
-{# TODO - simplify all these figures #}
+<!-- An image the browser assigns "High" priority, but we don't actually want that. -->
+<img src="/img/in_view_but_not_important.svg" importance="low" alt="I'm not important!">
+```
+
+With `preload` and `prefetch`, the priority is set by the browser depending on the type of resource. By using Priority Hints we can force the browser to change the default option.
 
 {{ figure_markup(
-  caption="The rate of priority hint adoption on mobile",
+  caption="The percent of Priority Hint adoption on mobile.",
   content="0.77%",
   classes="big-number",
   sheets_gid="1596669035",
   sql_file="priority_hints.sql"
 ) }}
 
-So far only 0.77% websites adopted this new hint as Chrome is still [actively](https://www.chromestatus.com/features/5273474901737472) experimenting, and at the time of this article's release the feature is on-hold.
-
-{{ figure_markup(
-  caption="Priority hints on mobile are on script elements",
-  content="80%",
-  classes="big-number",
-  sheets_gid="800402946",
-  sql_file="priority_hints_by_element.sql"
-) }}
+So far only 0.77% websites adopted this new hint as Chrome is still [actively](https://www.chromestatus.com/features/5273474901737472) experimenting, and at the time of this article's release the feature is on-hold. 
 
 The largest use is with script elements, which is unsurprising as the number of JS primary and third-party files continues to grow.
 
+{# TODO(authors): Make sure the query link is correct, and that the gid is correct. #}
```

Comment:
@Zizzamia

Reply:
The gid is correct (`1098063134`), and the I think this is the right query https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/21_Resource_Hints/priority_hints_by_importance.sql as mentioned in the figure. Right @khempenius @rviscomi ?